components/dev-tools/spack/SPECS/spack.spec

```diff
@@ -25,7 +25,6 @@ Source0:	https://github.com/LLNL/%{pname}/archive/v%{version}.tar.gz
 BuildRequires: rsync
 BuildRequires: python3
 Requires: bash
-Requires: curl
```

Comment:
Was this intentional for the CI processing update?

Reply:
I was looking for something to test first. But this is an actual fix. That line exists twice.